pages/docs/concepts/server.md

```diff
@@ -0,0 +1,15 @@
+
+## What is a server?
+A server is a representation of a messaging broker system where connections and communication between a producer and a consumer is established. Unlike traditional API servers which are depended on request/response, message brokers interactions occur back and forth over different TCP/IP.
+
+## What is the purpose of servers?
+Servers play an important role as they maintain a relationship between producers and consumers. When designing and setting up an event-driven application, servers are in charge of delivering asynchronous messages from the producer to the consumer through the use of channels. With the integration of different messaging protocols, servers can transmit and exchange messages between server-client connections or server to server connections.
+
```

Comment:
for broker centric might be better to do it this way

![IMG_20220704_185811](https://user-images.githubusercontent.com/6995927/177195858-5e780ff1-61ca-40ec-b687-d30d2eaca050.jpg)

or just add the same channels between server. As now it can be confused that messages are not sent to channels. But this is how it works, Producer sends to specific channel, and Subscriber listens to it

Reply:
@derberg done. I have made the suggested changes